docs/dagster-university/pages/dagster-dbt/lesson-3/3-representing-the-dbt-project-in-dagster.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'Lesson 3: Representing the dbt project in Dagster'
+module: 'dagster_dbt'
+lesson: '3'
+---
+
+# Representing the dbt project in Dagster
+
+As you’ll frequently point your Dagster code to the `target/manifest.json` file and your dbt project in this course, it’ll be helpful to keep a reusable representation of the dbt project. This can be easily done using the `DbtProject` class.
+
+Create a new file in the `assets` directory called `dbt.py`. Open that file and add the following import at the top:
```

Comment:
```suggestion
In the `assets` directory, create a new `dbt.py` file and add the following imports:
```

Reply:
This is what I see now: 

> In the `dagster_university` directory, create a new `project.py` file and add the following imports:

But later on in the lesson, `dbt.py` is still used (ex: line 31).